packages/dapp/package.json

```diff
@@ -20,7 +21,9 @@
     "@transferto/shared": "*",
     "@types/react": "^18.2.15",
     "@widgetbot/react-embed": "^1.7.0",
+    "contentful": "^10.2.3",
     "ethers": "^5.7.2",
+    "graphql": "^16.7.1",
```

Comment:
We added around 350kb with these packages, and I'm curious if anyone would like to optimize and make this feature with dynamic import? 🙂